examples/models/llama2/quantize.py

```diff
@@ -131,507 +130,6 @@ def dynamically_quantize_per_channel(
-    "int quant_max, ScalarType dtype, int group_size) -> Tensor"
-)
-
-
-# TODO: dtype is ignored for now
-@impl(
-    quantized_decomposed_lib, "quantize_per_channel_group", "CompositeExplicitAutograd"
-)
-def quantize_per_channel_group(
-    input: torch.Tensor,
-    scales: torch.Tensor,
-    zero_points: torch.Tensor,
-    quant_min: int,
-    quant_max: int,
-    dtype: torch.dtype,
-    group_size=128,
-):
-    assert group_size > 1
-    # needed for GPTQ single column quantize
-    if group_size > input.shape[-1] and scales.shape[-1] == 1:
-        group_size = input.shape[-1]
-
-    assert input.shape[-1] % group_size == 0
-    assert input.dim() == 2
-
-    # TODO: check for dtype, currently we can't express torch.int4 so it's omitted
-    to_quant = input.reshape(-1, group_size)
-    assert torch.isnan(to_quant).sum() == 0
-
-    scales = scales.reshape(-1, 1)
-    zero_points = zero_points.reshape(-1, 1)
-
-    input_int8 = (
-        to_quant.div(scales)
-        .add(zero_points)
-        .round()
-        .clamp_(quant_min, quant_max)
-        .to(dtype)
-        .reshape_as(input)
-    )
-
-    return input_int8
-
-
-@impl(quantized_decomposed_lib, "quantize_per_channel_group", "Meta")
-def quantize_per_channel_group_meta(
-    input: torch.Tensor,
-    scales: torch.Tensor,
-    zero_points: torch.Tensor,
-    quant_min: int,
-    quant_max: int,
-    dtype: torch.dtype,
-    group_size=128,
-):
-    """Groupwise quantization within each channel for an 2-d Tensor using the quantization parameters
-    to map from floating point to quantized values. This means for each row of a 2-d Tensor
-    (M, N), we calculate scales/zero_points for each `group_size` elements
-    and quantize every `group_size` elements with the same quantization parameter.
-    The dimension for scales/zero_points will be (M * ceil(N, group_size),)
-
-    Args:
-       input (torch.Tensor): original float32 or bfloat16 Tensor
-       scales (float32 torch.Tensor): quantization parameter for per channel group affine quantization
-       zero_points (int32 torch.Tensor): quantization parameter for per channel group affine quantization
-       quant_min (int): minimum quantized value for output Tensor
-       quant_max (int): maximum quantized value for output Tensor
-       dtype (torch.dtype): requested dtype (e.g. torch.uint8) for output Tensor
-
-    Returns:
-       Tensor with requested dtype (e.g. torch.uint8), note the quantization parameters
-       are not stored in the Tensor, we are storing them in function arguments instead
-    """
-    assert group_size > 1
-    # needed for GPTQ single column quantize
-    if group_size > input.shape[-1] and scales.shape[-1] == 1:
-        group_size = input.shape[-1]
-
-    assert input.shape[-1] % group_size == 0
-    assert input.dim() == 2
-    return torch.empty_like(input, dtype=dtype)
-
-
-def group_quantize_tensor_symmetric(
```

Comment:
deleted here, but still used by prepare_int4_weight_and_scales_and_zeros

Reply:
this is probably not used, we'll cleanup a bit later